### DISCUSSION TITLE
Unable to install Tailwind without PostCSS

### DISCUSSION THREAD

**Discussion by paridhi7** [tailwindlabs/tailwindcss#3651] [Help]
I'm following the [docs for installing Tailwind without PostCSS](https://tailwindcss.com/docs/installation#using-tailwind-without-post-css). When I run this command on my terminal: `npx tailwindcss-cli@latest build -o tailwind.css` I get the following error:
```npm ERR! code E405
npm ERR! 405 Method Not Allowed - GET https://registry.npmjs.org/tailwind-cli/tailwindcss-cli

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/dell/.npm/_logs/2021-02-25T11_18_02_718Z-debug.log
Install for tailwindcss-cli@latest failed with code 1```

My operating system is Ubuntu 20.04.1 LTS. Can somebody please help me with this?

**Comment by simonswiss** [comment]
Hey!

What version of node/npm are you using?

**Reply by paridhi7** [reply]
node version: v10.19.0
npm version: 6.14.4

**Reply by adamwathan** [reply]
The error looks like a temporary network error but you'll also need Node v12+ to use Tailwind.
